lib/phobos.rb

```diff
@@ -77,16 +78,20 @@ def configure_logger
         appenders << Logging.appenders.file(log_file, layout: json_layout)
       end
 
-      @ruby_kafka_logger = nil
+      @ruby_kafka_logger = config.custom_kafka_logger
 
-      if ruby_kafka
+      if ruby_kafka && @ruby_kafka_logger.nil?
```

Comment:
to be in line with the newly added block below, perhaps do

```
if config.custom_kafka_logger
  @ruby_kafka_logger = config.custom_kafka_logger
else
  ...
end
```

Reply:
sure!